src/test/java/org/takes/tk/TkProxyTest.java

```diff
@@ -57,15 +171,46 @@ public void exec(final URI home) throws IOException {
                         new RsPrint(
                             new TkProxy(
                                 String.format(
-                                    "%s:%d",
-                                    home.getHost(), home.getPort()
-                                )
+                                    FORMAT, home.getHost(), home.getPort()
+                                    )
                             ).act(new RqFake())
                         ).print(),
-                        Matchers.containsString("hello")
-                    );
+                            Matchers.allOf(
+                                        Matchers.containsString(HELLO_WORLD),
+                                        Matchers.containsString(
+                                                HEADER
```

Comment:
@aschworer `HEADER` is used only here, you can inline it and remove static field